subsys/bluetooth/host/hci_core.h

```diff
@@ -30,7 +30,7 @@ enum {
 	BT_DEV_ENABLE,
 	BT_DEV_READY,
 	BT_DEV_PRESET_ID,
-	BT_DEV_USER_ID_ADDR,
+
```

Comment:
Any particular reason you're changing the grouping here by adding an empty line?

Reply:
Didn't mean to do that, reverted